### PR TITLE
Add proper data type handling

### PR DESCRIPTION
**Pull request by OskarDamkjaer**

### CODE REVIEW

File: src/neo4j-arc/common/utils/driverUtils.ts [hunk 1]
```diff
@@ -0,0 +1,137 @@
...
+        item.properties,
+        getPropertyTypeDisplayName
+      )
+    }
+  })
+
+  const relationships = Array.from(relMap.values())
+    .filter(item => {
+      // We can't display dangling relationships such as
+      // match ()-[a:ACTED_IN]->() return a;
```
Comment (eijawerner) [inline src/neo4j-arc/common/utils/driverUtils.ts:117]: 👍

**Review by eijawerner — COMMENTED** [review]
Really nice! ⚡ I think we need more tests on this though. And look over the TODO:s left.

**Review by eijawerner — APPROVED** [review]
Ok, LGTM when all tests are green.

**Comment by OskarDamkjaer** [comment]
solved the yarn.lock merge :+1:

**Pull request merged**